src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -215,9 +215,11 @@ method net.sf.json.JSON size
 staticMethod org.codehaus.groovy.runtime.DateGroovyMethods format java.util.Date java.lang.String
 staticMethod org.codehaus.groovy.runtime.DateGroovyMethods format java.util.Date java.lang.String java.util.TimeZone
 staticMethod org.codehaus.groovy.runtime.DateGroovyMethods getDateTimeString java.util.Date
+
```

Comment:
???

Reply:
I believe this newline separates the DateGroovyMethods from the DefaultGroovyMethods.  Spacing in other places of the file seemed kind of arbitrary https://github.com/jenkinsci/script-security-plugin/blob/2f466190d4030d0e36025fdcab53d88323441114/src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist#L17

Reply:
I will go ahead with another commit to take this newline out, because I would love for this to be merged without any qualms.